### PR TITLE
feat(camemberts): #103/#104 move charts above results table

### PR DESCRIPTION
**Pull request by plaplante-lab**

### CODE REVIEW

File: src/packages/@cqdg/components/charts/index.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ export const ColumnCenter = styled(Column, {
 export const WrappedRow = styled(Row, {
   alignItems: 'center',
   flexWrap: 'wrap',
-  justifyContent: 'flex-start',
+  justifyContent: 'flex-start'
```
Comment (francisl) [inline src/packages/@cqdg/components/charts/index.js]: Can you put that in a css file and delete the `styled` if that can be done quickly?

**Review by francisl — COMMENTED** [review]
LGTM, one comment about moving stuff in `.css` files

File: src/packages/@cqdg/components/charts/index.js (hunk 1 above)
Comment (plaplante-lab) [reply src/packages/@cqdg/components/charts/index.js]: Done.

**Review by francisl — APPROVED** [review]
:shipit:

**Pull request merged**